### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by axecopfire**

### CODE REVIEW

**Comment by axecopfire** [comment]
we can have a convo

**Comment by CyberCityCircuits** [comment]
Neat

**Review by CyberCityCircuits — CHANGES_REQUESTED** [review]
Doesn't work.

**Comment by axecopfire** [comment]
Updated pic to have 38 px or whatever width

**Review by CyberCityCircuits — CHANGES_REQUESTED** [review]
Too Tiny!

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 *Hello!*  Welcome to the DREAMWEAVER 4N repo.  The DREAMWEAVER 4N is an Open Source 4 Channel Sequencer or Arpeggiator based on the Arduino architecture.  This is something that I have been working on for several months.  If you have any comments or suggestions, I can be reached on Twitter [@MakeAugusta](https://twitter.com/MakeAugusta).
 
-![DW4N 1.6 Beta](https://github.com/CyberCityCircuits/Dreamweaver_4N/blob/master/Images/1_6_Beta_001.jpg)
+<img src ="https://github.com/CyberCityCircuits/Dreamweaver_4N/blob/master/Images/1_6_Beta_001.jpg" alt="DW4N 1.6 Beta" width="45">
```
Comment (axecopfire) [inline README.md]: Too Tiny

Comment (CyberCityCircuits) [inline README.md]: This shows up too small.

**Review by CyberCityCircuits — APPROVED** [review]

**Pull request merged**